android/src/main/kotlin/be/tramckrijte/workmanager/EchoingWorker.kt

```diff
@@ -29,7 +29,7 @@ class EchoingWorker(private val ctx: Context,
         const val IS_IN_DEBUG_MODE = "be.tramckrijte.workmanager.IS_IN_DEBUG_MODE"
 
         const val BACKGROUND_CHANNEL_NAME = "be.tramckrijte.workmanager/background_channel_work_manager"
-        const val BACKGROUND_CHANNEL_INITIALIZED = "backgroundChannelInitialized"
+        const val BACKGROUND_CHANNEL_INITIALIZED = "backgroundJobDidComplete"
```

Comment:
Even thought the name is completely internal so it does not really matter what this is called, I think it implies the implementer code was already done executing while it has merely started the background channel. 
We would still need to send the execution result back on this channel.